service/controller/app.go

```diff
@@ -0,0 +1,120 @@
+package controller
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/client/k8scrdclient"
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/informer"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/app-operator/service/controller/v1"
+)
+
+type AppConfig struct {
```

Comment:
```suggestion
 type Config struct {
```

Reply:
This is outdated.

Reply:
@rossf7 You mean we don't need to change it as `Config`?

Reply:
Sorry I mean we should rename to `Config`. `AppConfig` is the old naming convention.